### PR TITLE
i18n: fixed & added russian translations

### PR DESCRIPTION
**Pull request by anmorgunov**
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.

Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.

### CODE REVIEW

File: i18n/ru.po [hunk 1]
```diff
@@ -3378,10 +3397,15 @@ msgid ""
 "Middle Mouse: \tClick and drag to zoom in or out.\n"
 "Right Mouse: \tClick and drag to move the view.\n"
 msgstr ""
+Инструмент изменения обзора\n"
```
Comment (ghutchis) [inline i18n/ru.po:3400]: Is this correct? It looks like there are un-matching " here.

File: i18n/ru.po [hunk 2]
```diff
@@ -3230,6 +3242,13 @@ msgid ""
 "\tDihedral is measured between 1-2-3-4\n"
 "Right Mouse: \tReset the measurements."
 msgstr ""
+Линейка\n"
```
Comment (ghutchis) [inline i18n/ru.po:3245]: It looks like there are un-matching " here.

File: i18n/ru.po [hunk 3]
```diff
@@ -2673,6 +2673,10 @@ msgid ""
 "Left Mouse: \tClick and Drag to create Atoms and Bond\n"
 "Right Mouse: \tDelete Atom"
 msgstr ""
+Инструмент построение молекулы\n"
```
Comment (ghutchis) [inline i18n/ru.po:2676]: It looks like there are un-matching " here.

**Review by ghutchis — COMMENTED** [review]
Thanks very much - looks like there are some minor issues with the .po file though. Are the "" matching?

File: i18n/ru.po (hunk 1 above)
Comment (anmorgunov) [reply i18n/ru.po]: Yes, that's a mistake. I've fixed in all three cases, thank you

**Comment by anmorgunov** [comment]
Added another commit with fixes

**Pull request merged**